### DISCUSSION TITLE
How to update the docker version to latest GitHub release?

### DISCUSSION THREAD

**Discussion by rpan9** [jellyfin/jellyfin#12212] [General]

**Comment by TheMelmacian** [comment]
Don't know if still relevant, since the question was closed, but here is the answer:

The docker image for the latest official stable release is `jellyfin/jellyfin:latest`

https://jellyfin.org/downloads/docker/
https://hub.docker.com/r/jellyfin/jellyfin/
https://jellyfin.org/docs/general/installation/container

---

If you search for the latest development state: `jellyfin/jellyfin:unstable`.
But this is intended for testing and not recommended for productive environments. It is not guaranteed that a rollback to a previous stable release is possible when unstable is used.
